DatadogSessionReplay/Sources/Drafts/SessionReplayFeature.swift

```diff
@@ -62,7 +62,9 @@ internal class SessionReplayFeature: DatadogFeature, SessionReplayController {
         self.requestBuilder = RequestBuilder(customUploadURL: configuration.customUploadURL)
         self.performanceOverride = PerformancePresetOverride(
             maxFileSize: UInt64(10).MB,
-            maxObjectSize: UInt64(10).MB
+            maxObjectSize: UInt64(10).MB,
+            meanFileAge: 5, // equivalent of `batchSize: .small` - see `DatadogCore.PerformancePreset`
+            minUploadDelay: 1 // equivalent of `uploadFrequency: .frequent` - see `DatadogCore.PerformancePreset`
```

Comment:
a little bit ugly. Maybe we could share those enums 🤔 

Is it any easier in v2?

Reply:
In a simple way we can't because it is defined in `DatadogCore` and SR doesn't depend on that module. It depends on `DatadogInternal` and moving this enum in there would further break the isolation (it's already partially broken due to `PerformancePresetOverride`). I think long term we rather won't need overrides and invent something better for this problem.

@maxep what's your take on this?

Reply:
I think settings by value enable greater flexibility. If that was exposed to user, then yes enum would have been best but since this is an internal contract between the core and Features, we can use values 👍